cdk/domino_cdk/config/iam.py

```diff
@@ -0,0 +1,399 @@
+# Future TODO item: Incorporate IAM reqs into the provisioning
```

Comment:
Based on recent reviews and confusion, I would greatly recommend to keep original policies in JSON files and just do replace here. 

It is extremely hard to understand what is going on here and I do not understand the value of this move of the simple config files into complex code.

Reply:
The original policies still live in those files for now. This generator code exists for the moment as a tool to unify the policies more. We'll probably have to split it into two policies so it can all fit, which should enable us to get very close to having a single policy (maybe just the "coming from cloudformation" part being the only variation). After that point, this should be a lot less confusing.